frontend/src/views/ShootList.vue

```diff
@@ -155,6 +194,15 @@ export default {
         this.$localStorage.removeItem(`project/${this.projectName}/shoot-list/options`) // clear project specific options
         this.$localStorage.setObject('projects/shoot-list/options', { sortBy, sortDesc, itemsPerPage })
       }
+    },
+    freezeSorting (value) {
+      if (value) {
+        const { sortBy, sortDesc } = this.options
+        const sortedItems = this.sortItems(this.items, sortBy, sortDesc)
+        this.sortedUIDsAtFreeze = map(sortedItems, 'metadata.uid')
```

Comment:
How is this related to the uids in the store? Why do we store the list of uids in two different locations?

Reply:
This is due to historic reasons when we had the sorting in the getter.
Right now with the current changes there are two lists of UIDs. One unsorted in the store that is set when the user enabled the focus mode. And one in the component to cache the last sorting order. Let's discuss if we want to move the sorted UIDs to the store, or do something else there are multiple options.